src/components/TableFilterSelection/index.js

```diff
@@ -51,49 +57,66 @@ export default class TableFilterSelection extends Component {
     this.setState({ updateFiltersDisabled: false });
   };
 
+  onToggle = isOpen => {
+    this.setState({
+      isOpen,
+    });
+  };
+
   render() {
     const { filters } = this.props;
-    const { selectedFilters, updateFiltersDisabled } = this.state;
+    const { selectedFilters, updateFiltersDisabled, isOpen } = this.state;
+
+    const dropdownItems = [
+      <DropdownItem style={{ width: '50vw' }}>
+        <div>
+          <Form className={styles.form}>
+            <Flex>
+              {Object.keys(filters).map(category => (
+                <FlexItem key={category} style={{ marginRight: 16, marginBottom: 16, width: 160 }}>
+                  <p style={{ marginBottom: 4, fontSize: 12, fontWeight: 600 }}>{category}</p>
+                  <Select
+                    key={category}
+                    selected={selectedFilters[category]}
+                    options={filters[category]}
+                    onSelect={(event, value) => this.onFilterChange(event, value, category)}
+                  />
+                </FlexItem>
+              ))}
+            </Flex>
+            <Button
+              variant="primary"
+              type="submit"
+              isDisabled={updateFiltersDisabled}
+              onClick={this.onFilterTable}
+            >
+              Save
+            </Button>
+            {/* </div> */}
```

Comment:
Is the commented `</div>` here inside empty brackets important somehow?

Reply:
Node, left it by by mistake. Resolved now.